#! /bin/sh /usr/share/dpatch/dpatch-run
## 03-gtk-includes.dpatch by  <martin.pitt@ubuntu.com>
##
## DP: Work around https://launchpad.net/bugs/247909 FTBFS

@DPATCH@
diff -urNad inkscape-0.46~/src/widgets/sp-xmlview-attr-list.h inkscape-0.46/src/widgets/sp-xmlview-attr-list.h
--- inkscape-0.46~/src/widgets/sp-xmlview-attr-list.h	2008-03-11 05:20:13.000000000 +0100
+++ inkscape-0.46/src/widgets/sp-xmlview-attr-list.h	2008-08-11 12:31:37.000000000 +0200
@@ -13,7 +13,7 @@
  */
 
 #include <stdio.h>
-#include <gtk/gtkclist.h>
+#include <gtk/gtk.h>
 #include "../xml/repr.h"
 
 #include <glib.h>
